py-polars/polars/series/binary.py

```diff
@@ -104,6 +104,8 @@ def decode(self, encoding: TransferEncoding, *, strict: bool = True) -> Series:
 
         Examples
         --------
+        Decode hex values to binary.
```

Comment:
I don't think "decode to" is proper English?

Reply:
Agree - will remove "to binary" from those lines.